### ISSUE TITLE
\mid is thicker than \vert

### ISSUE THREAD

**Issue by xsrvmy** [stipub/stixfonts#191]
I have noticed that \mid (and by extension \nmid) are thicker than \vert. Not sure if this inconsistency is intended or if it is a bug. It only becomes a problem when you want an extended \mid because you have to use \bigm| and that produces a thinner bar than \mid.

**Comment by tiroj** [comment]
I should note, in regard to this and other feedback, that I and the other font makers working on the STIX fonts are not (La)Tex users, so will be reliant on others to explain or translate some issues so that we can better understand how they relate to the internal font data.

**Comment by davidmjones** [comment]
>  will be reliant on others to explain or translate some issues so that we can better understand how they relate to the internal font data.

@tiroj Yes, I'm making a note of these and will provide examples and further information once we start to work on the MATH table.
